blockchain/chain.go

```diff
@@ -880,22 +889,16 @@ func countSpentOutputs(block *btcutil.Block) int {
 //
 // This function may modify node statuses in the block index without flushing.
 //
+// This function never leaves the utxo set in an inconsistent state for block
+// disconnects.
+//
```

Comment:
Stray empty comment line here.

Reply:
It's kinda nice to separate this from

`// This function MUST be called with the chain state lock held (for writes).`

right? Since that's the format used in other parts of the code as well.

Though I can get rid of it if you think it's better

Reply:
Ah nvm, when I looked at the diff originally, didn't see the comment below.